canonical/checkbox
# Documentation migration

## Description



The goal of this PR is to migrate our existing pages to the new `docs` directory, and do some cleanup. Namely:

- Update the welcome page with a description of the application
- Copy *.rst files from checkbnox-ng/docs to docs/ following the proposed [structure](https://docs.google.com/spreadsheets/d/1RiCawU15NYpcC85_w444ZSHOAVHWE40XQEy8koz8aUU/edit#gid=0)
- Remove the starter pack boilerplate

I migrated everything except `snappy.rst` as I thought it was not relevant anymore (checkbox-snappy is not in use, and the content of that page is basically a mega-tutorial with information present elsewhere).

## Resolved issues



CHECKBOX-610

## Tests

Tested locally using `make run` (super useful stuff!)

I checked if images and links worked as before, and it seems OK.

To test locally:

1. Checkout this branch
2. Go to the `docs/` directory, and run `make install`
3. Run `make run` -> you should be able to access the built documentation locally.
